core/chainio/avs_writer.go

```diff
@@ -126,7 +136,8 @@ func (w *AvsWriter) SendAggregatedResponse(batchIdentifierHash [32]byte, batchMe
 			return nil, err
 		}
 
-		receipt, err := utils.WaitForTransactionReceiptRetryable(w.Client, w.ClientFallback, tx.Hash(), timeToWaitBeforeBump)
+		waitForTxConfig.MaxElapsedTime = timeToWaitBeforeBump
```

Comment:
Better move this to the beginning of the function where you define the rest of the config so it is all in one place.

Reply:
moved!